src/lua/image.c

```diff
@@ -525,6 +525,10 @@ int dt_lua_init_image(lua_State *L)
   luaA_struct_member(L, dt_image_t, exif_maker, char_64);
   luaA_struct_member(L, dt_image_t, exif_model, char_64);
   luaA_struct_member(L, dt_image_t, exif_lens, char_128);
+  luaA_struct_member(L, dt_image_t, exif_whitebalance, char_64);
+  luaA_struct_member(L, dt_image_t, exif_flash, char_64);
```

Comment:
dt_lua_image_t is a subset of dt_image_t in src/common/image.h, so this wouldn't be the right place for the icon.

Where is the flash icon defined?

Currently the variable replacement functions in Lua don't support graphical elements, because at the time I was using it as a file name/path constructor.  Looking at the bigger picture, I could see where this would be useful in other scenarios so I need to add support for the graphical elements too, but in a separate PR.

Reply:
The "icon" is just a Unicode string with a symbol character sequence.  Same as the RATING.ICONS and LABELS.ICONS variables (though the latter also includes HTML markup).

Reply:
I just saw the conversation in #17501, so now I get it.  I'll implement the flash icon in the substitution library.  Thanks